e2e/iam/orgs_api_keys_test.go

```diff
@@ -33,15 +33,14 @@ const (
 	apiKeysEntity = "apikeys"
 )
 
-func TestOrgAPIKeys(t *testing.T) {
+func TestOrgsAPIKeys(t *testing.T) {
```

Comment:
[nit-grammar] only pluralize the last word, same for the file name
```suggestion
func TestOrgAPIKeys(t *testing.T) {
```